text-recognition/index.d.ts

```diff
@@ -57,14 +57,33 @@ export interface TextRecognitionResult {
   blocks: TextBlock[];
 }
 
+export enum TranslateRecognitionScript {
```

Comment:
I think it should be `TextRecognitionScript`.

```suggestion
export enum TextRecognitionScript {
```